src/main/java/duke/command/ExitCommand.java

```diff
@@ -0,0 +1,27 @@
+package duke.command;
+
+import duke.Storage;
+import duke.TaskList;
+import duke.Ui;
+
+/**
+ * ExitCommand closes the Duke chatbot.
+ */
+public class ExitCommand extends Command {
+
+    /**
+     * Constructor for ExitCommand.
+     */
+    public ExitCommand() {
+        super();
+        this.isExit = true;
```

Comment:
Not sure if I missed it, but `Duke.java` does not seem to be checking the value of `isExit` after `command.execute()`. So is there a purpose for setting `isExit = true`?